apps/omg_performance/lib/omg_performance/byzantine_events.ex

```diff
@@ -0,0 +1,112 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Performance.ByzantineEvents do
+  @moduledoc """
+  OMG network child chain server byzantine event test entrypoint. Setup and runs performance byzantine tests.
+
+  # Usage
+
+  See functions in this module for options available
+
+  ## start_dos_get_exits runs a test to get exit data for given 10 positions for 3 users
+
+  ```
+  mix run --no-start -e \
+    '
+      OMG.Performance.ByzantineEvents.Generators.stream_utxo_positions() |>
+      Enum.take(10) |> OMG.Performance.ByzantineEvents.start_dos_get_exits(3)
+    '
+  ```
+
+  __ASSUMPTIONS:__
+  This test should be run on testnet filled with transactions make sure you followed instructions in `docs/demo_05.md`
+  and `geth`, `omg_child_chain` and `omg_watcher` are running; and watcher is fully synced.
+
+  Expected result of running the above command should looks like:
+
+  ```
+  [
+    %{time: 232000, corrects_count: 10, errors_count: 0},
+    %{time: 221500, corrects_count: 10, errors_count: 0},
+    %{time: 219900, corrects_count: 10, errors_count: 0},
+  ]
+  ```
+
+  where time is expressed in microseconds and the sum of `corrects_count + errors_count` should equal to
```

Comment:
Let's retain the conventions from the original perftest: `time` => `span_ms` and give the time in miliseconds instead

Reply:
I see this got resolved but still the doc is out of date